addons/web/static/tests/core/l10n/dates_tests.js

```diff
@@ -154,6 +154,34 @@ QUnit.module(
             );
         });
 
+        QUnit.test("parseDate with different numbering system", async (assert) => {
+            patchWithCleanup(localization, {
+                dateFormat: "dd MMM, yyyy",
+                dateTimeFormat: "dd MMM, yyyy hh:mm:ss",
+                timeFormat: "hh:mm:ss",
+            });
+
+            patchWithCleanup(Settings, { defaultNumberingSystem: "arab", defaultLocale: "ar" });
+
+            assert.equal(parseDate("٠١ فبراير, ٢٠٢٣").toISO(), "2023-02-01T00:00:00.000+01:00");
+        });
+
+        QUnit.test(
+            "parseDate where the time appears before the date in the format",
+            async (assert) => {
+                patchWithCleanup(localization, {
+                    dateFormat: "dd MM yyyy",
+                    dateTimeFormat: "hh:mm:ss dd MM yyyy",
```

Comment:
The datetime format is not computed with the time before the date, and should never be.

See: https://github.com/odoo/odoo/blob/11f44de490d7e9c59b76aaade1d31b3041c9fb3b/addons/web/static/src/core/l10n/localization_service.js#L76

Reply:
Thanks, I made the change.

Reply:
My bad, I didn't see that